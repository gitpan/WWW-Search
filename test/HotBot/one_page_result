http://www.hotbot.com/text/default.asp?SM=SC&MT=%22Christie+Abbott%22&DE=2&DC=100
