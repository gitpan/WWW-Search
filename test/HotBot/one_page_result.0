
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">

<html><head>
<title>HotBot Results: "Christie Abbott" (1+)</title></head>
<FORM ACTION="/text/default.asp">
<font face=Arial,Geneva,sans-serif><b>HotBot: Text-only version</b> | <a href="/
?act.revise=+Revise+&MT=%22Christie+Abbott%22&SM=SC&DC=100
">Full graphics version</a>
<p>

<b>look for:</b>&nbsp;&nbsp;</FONT><SELECT name="SM" >
<OPTION value="MC">all the words
<OPTION value="SC" selected>any of the words
<OPTION value="phrase">exact phrase
<OPTION value="title">the page title
<OPTION value="name">the person
<OPTION value="url">links to this URL
<OPTION value="B">Boolean phrase
</SELECT>
<INPUT type=text name="MT" value="&quot;Christie Abbott&quot;" size=30 maxlength=300>
&nbsp;
<input type="submit" name="search" value="Search"><br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<input type="SUBMIT" name="act.new" value=" New ">&nbsp;<input type="SUBMIT" name="act.revise" value=" Revise ">&nbsp;

<input type="SUBMIT" name="act.super" value=" More Options ">

<font face=Arial,Geneva,sans-serif>

<!-- ADS  -->
<p>
<!-- AdSpace program=hotbotresults size=large params.Styles=html,imgtxt,img,text SERVER_NAME=www.hotbot.com KW=christie KW=abbott KW=christie%20abbott KW=christie%20abbott ALL_KW=christie%20abbott cobrand= REMOTE_HOST=dash.isi.edu RH1=edu RH2=isi.edu RH3=dash.isi.edu REMOTE_ADDR=128.9.160.184 HOT_R_UNIQID=28GsWd5njCQG P_UNIQID=28GsWd5njCwG hbf=adult hbf=language:any --><a href="http://nsads.hotwired.com/event.ng/Type=click&ProfileID=5576&RunID=10568&AdID=12769&GroupID=1&FamilyID=1227&TagValues=1.6.25.145.156.258.389.409.526.539.4659.4673.4693.4724.4837.4863.6456.6627.6820.6830.69547&params.tag.ALL_KW=christie+abbott&Redirect=http:%2F%2Fad.preferences.com%2Fclick;spacedesc%3DGlobalBrandingQ4_Hotbot_468x60_RunOfSite_Any%26time%3D1998.12.10.18.29.21.0"><img src="http://ad.preferences.com/image;spacedesc=GlobalBrandingQ4_Hotbot_468x60_RunOfSite_Any&time=1998.12.10.18.29.21.0
" border=1 height=60 width=468 alt="Compaq. Better Answers. Click Here."><br>Compaq. Better Answers. Click Here.</a><!-- /AdSpace -->
<!-- END ADS -->

<p>
<blockquote>

<b>Returned:&nbsp;
15 matches.</b><br>
<!--
<b>Breakdown:</b>&nbsp;<p>
-->

	<!-- foo -->


<b>Results&nbsp;1&nbsp;-&nbsp;15&nbsp;</b>

<p>
<!-- BRES -->
<p>


<B>1. </B><A HREF="http://www.metroplexweb.com/actingclass/">Margo Manning Acting School</A><BR>Progressive acting classes, drama and theatrical instruction.  Located in Dallas, Texas.<br>99%&nbsp;&nbsp;&nbsp; 7361 bytes&#44; 1998/08/28&nbsp;&nbsp;&nbsp;http://www.metroplexweb.com/actingclass/<p>
<B>2. </B><A HREF="http://www.palladiumnet.com/WWWishbone/MyShow/TheCast/">WORLDWIDE WISHBONE(TM)</A><BR>&quot;Meet the members of my pack!&quot; The WISHBONE television series stars Wishbone the dog, &quot;a little dog with a big imagination.&quot;SM The Wishbone character is played by a real Jack Russell terrier (with a voice-over by actor Larry Brantley), who &quot;sniffs..<br>96%&nbsp;&nbsp;&nbsp; 4392 bytes&#44; 1997/04/15&nbsp;&nbsp;&nbsp;http://www.palladiumnet.com/WWWishbone/MyShow/TheCast/<p>
<B>3. </B><A HREF="http://crisny.org/not-for-profit/cllassie/12tteamb.htm">12 &amp; Under B Classie Lassie Tournament Team</A><BR>12 &amp; Under 'B' Classie Lassie Tournament Team Team Roster Christie Abbott Trisha Baldwin Ruth Brown Megan Curran Kelly Ferguson Nicole Gorman Corey Lindley Casy Maddalla Jennifer Pfeffer Jessica Sickles Amanda Stan Jenah Vielkind Ashley Zeller...<br>93%&nbsp;&nbsp;&nbsp; 1557 bytes&#44; 1997/10/21&nbsp;&nbsp;&nbsp;http://crisny.org/not-for-profit/cllassie/12tteamb.htm<p>
<B>4. </B><A HREF="http://www.sagami.or.jp/~harasawa/seiyuu/List/NishimuraChinami.html">Katayori seiyuu list : NishimuraChinami</A><BR>index sapporo.iij.ad.jp seiyuu database () WISHBONE : ``FURST IMPRESSIONS&quot; 1998/10/03 1998/10/01 # () # ^^; WISHBONE : ``THE PAWLOINED PAPER&quot; 1998/09/12 62 VS 1998/09/10 WISHBONE : ``THE CANINE CURE&quot; 1998/09/05 4 57 ! 1998/08/06 WISHBONE : ``A TALE.<br>93%&nbsp;&nbsp;&nbsp; 14096 bytes&#44; 1998/10/03&nbsp;&nbsp;&nbsp;http://www.sagami.or.jp/~harasawa/seiyuu/List/NishimuraChinami.html<p>
<B>5. </B><A HREF="http://pbs.org/wishbone/factsabouttheseries.html">WISHBONE - Facts About the Series</A><BR>FACTS ABOUT THE WISHBONE SERIES PREMIERE DATE WISHBONE premiered on PBS on Oct. 9, 1995. Forty half-hour WISHBONE episodes will air daily on some 300 PBS stations over a three-year period. CONCEPT The WISHBONE educational television series airing...<br>91%&nbsp;&nbsp;&nbsp; 3181 bytes&#44; 1997/06/10&nbsp;&nbsp;&nbsp;http://pbs.org/wishbone/factsabouttheseries.html<p>
<B>6. </b> Mirror: <a href=http://www.pbs.org/wishbone/factsabouttheseries.html>&#40;http://www.pbs.org/wishbone/factsabouttheseries.html&#41;</A>
<B>7. </B><A HREF="http://www.webmill.com/WebSleuth/celeb/adam_springfield.shtml">'adam springfield'</A><BR>What do you want to know today? &amp;trade; Have a question? Ask the WebSleuth at websleuth@webmill.com. Or visit Prompt Software's WebSleuth . . Please support our sponsors of/WebSleuth/celeb/adam_springfield.shtml Achieve Your Dreams Visit The Web...<br>90%&nbsp;&nbsp;&nbsp; 13529 bytes&#44; 1998/08/26&nbsp;&nbsp;&nbsp;http://www.webmill.com/WebSleuth/celeb/adam_springfield.shtml<p>
<B>8. </B><A HREF="http://www3.pbs.org/whatson/1997/10/press/wish__000201.html">October 1997 PBS Picks - WISHBONE</A><BR>WISHBONE&amp;#174 WISHBONE, the literature-loving Jack Russell terrier, bounds into the PBS fall season with a primetime special and new half-hour episodes of the series designed to introduce children to the great classics of world literature and help..<br>89%&nbsp;&nbsp;&nbsp; 5288 bytes&#44; 1997/10/28&nbsp;&nbsp;&nbsp;http://www3.pbs.org/whatson/1997/10/press/wish__000201.html<p>
<B>9. </B><A HREF="http://www.hollywood.com/news/topstories/06-09-98/html/2-5.html">Hollywood Online - Top Stories</A><BR>Tuesday, June 9 1998 Weekly Video Calendar L.A. Times Syndicate The following videocassettes are slated for release the week of June 7, 1998. Unless otherwise noted, all titles are released on Tuesdays: DRAMA -- AFTERGLOW (Columbia TriStar, priced..<br>89%&nbsp;&nbsp;&nbsp; 9788 bytes&#44; 1998/08/22&nbsp;&nbsp;&nbsp;http://www.hollywood.com/news/topstories/06-09-98/html/2-5.html<p>
<B>10. </B><A HREF="http://149.48.46.5/insidepbs/news/wishbone2.html">&quot;Wishbone&quot; Bounds Into PBS Prime Time This Fall With An Hour-Long Season Premiere Based On Washington Irving's The</A><BR>&quot;WISHBONE&quot; BOUNDS INTO PBS PRIME TIME THIS FALL WITH AN HOUR-LONG SEASON PREMIERE BASED ON WASHINGTON IRVING'S THE LEGEND OF SLEEPY HOLLOW - Eight New Half-Hour Episodes to Follow - RICHARDSON, TX, April 8, 1997 -- WISHBONE, the highly acclaimed...<br>86%&nbsp;&nbsp;&nbsp; 7066 bytes&#44; 1997/06/10&nbsp;&nbsp;&nbsp;http://149.48.46.5/insidepbs/news/wishbone2.html<p>
<B>11. </B><A HREF="http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/WWW/Search/HotBot.html">WWW::Search::HotBot - class for searching HotBot</A><BR>Contained in WWW-Search-1.022.tar.gz requires perl 5.004 or later NAME SYNOPSIS DESCRIPTION OPTIONS Format / Treatment of Query Terms Restricting Search to a Date Range Restricting Search to a Geographic Area Requesting Certain Multimedia Data...<br>84%&nbsp;&nbsp;&nbsp; 13861 bytes&#44; 1998/12/04&nbsp;&nbsp;&nbsp;http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/WWW/Search/HotBot.html<p>
<B>12. </B><A HREF="http://ring.aist.go.jp/archives/lang/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm">&#40;http://ring.aist.go.jp/archives/lang/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm&#41;</A><BR>#!/usr/local/bin/perl -w # HotBot.pm # by Wm. L. Scheding and Martin Thurn # Copyright (C) 1996-1998 by USC/ISI # $Id: one_page_result.0,v 1.9 1998/12/11 23:08:47 johnh Exp $ package WWW::Search::HotBot; =head1 NAME WWW::Search::HotBot - class for...<br>78%&nbsp;&nbsp;&nbsp; 12523 bytes&#44; 1998/08/11&nbsp;&nbsp;&nbsp;http://ring.aist.go.jp/archives/lang/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm<p>
<B>13. </B><A HREF="http://www.kfki.hu/ftp/packages/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm">&#40;http://www.kfki.hu/ftp/packages/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm&#41;</A><BR>#!/usr/local/bin/perl -w # HotBot.pm # by Wm. L. Scheding and Martin Thurn # Copyright (C) 1996-1998 by USC/ISI # $Id: one_page_result.0,v 1.9 1998/12/11 23:08:47 johnh Exp $ package WWW::Search::HotBot; =head1 NAME WWW::Search::HotBot - class for...<br>78%&nbsp;&nbsp;&nbsp; 12523 bytes&#44; 1998/08/11&nbsp;&nbsp;&nbsp;http://www.kfki.hu/ftp/packages/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm<p>
<B>14. </B><A HREF="http://www.gbgm-umc.org/willowhill/news/newslet/newslet.html">News Letter August</A><BR>Willow News WILLOW HILL UNITED METHODIST CHURCH MONTHLY NEWSLETTER Rev. E. Paul Unger, Pastor Germantown Hills, IL&amp;#9;&amp;#9;August 1998 EDITION&amp;#9;&amp;#9;Volume 1, No. 20 WHAT S INSIDE Facts &amp; Figures Pastor s Page Calendar of Events Birthdays ...<br>77%&nbsp;&nbsp;&nbsp; 30085 bytes&#44; 1998/07/30&nbsp;&nbsp;&nbsp;http://www.gbgm-umc.org/willowhill/news/newslet/newslet.html<p>
<B>15. </B><A HREF="http://www.lumal.com/usa/cis/female.html">July 14, 1998 12:24:53</A><BR>BACK TO THE GATE&gt; Agent unknown Ai Wan Aida Turturro Alanis Morissette Alex Meneses Alexandra Morgan Alison LaPlaca Alison Leigh MIlls Alison Porter l Allison Dunbar Allison Folland Amanda Anka l Amanda Boyington Amanda McBroom Amy Pietz Andrea...<br>61%&nbsp;&nbsp;&nbsp; 117793 bytes&#44; 1998/07/15&nbsp;&nbsp;&nbsp;http://www.lumal.com/usa/cis/female.html<p>
<!-- ERES -->

<INPUT type=hidden name="_v" value="2" >
<INPUT type=hidden name="OPs" value="MDRTP" >

<INPUT type=hidden name="DC" value="100" >
</form>

<form action="/text/default.asp">

</blockquote>
<b>look for:</b>&nbsp;&nbsp;<SELECT name="SM" >
<OPTION value="MC">all the words
<OPTION value="SC" selected>any of the words
<OPTION value="phrase">exact phrase
<OPTION value="title">the page title
<OPTION value="name">the person
<OPTION value="url">links to this URL
<OPTION value="B">Boolean phrase
</SELECT>
<INPUT type=text name="MT" value="&quot;Christie Abbott&quot;" size=30 maxlength=300>
&nbsp;
<input type="submit" name="search" value="Search"><br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<input type="SUBMIT" name="act.new" value=" New ">&nbsp;<input type="SUBMIT" name="act.revise" value=" Revise ">&nbsp;

<input type="SUBMIT" name="act.super" value=" More Options ">
<INPUT type=hidden name="_v" value="2" >
<INPUT type=hidden name="OPs" value="MDRTP" >

<INPUT type=hidden name="DC" value="100" >
</form>

<p><font size=1>

 

<!-- textfoot.asp -->
<FONT face="Verdana, Arial, Geneva" SIZE="-1">
<a href="http://www.wired.com/home/copyright.html">Copyright</a> &copy; 1994-98 Wired Digital Inc. All rights reserved.<BR></FONT>
<!-- /textfoot.asp -->


</html>