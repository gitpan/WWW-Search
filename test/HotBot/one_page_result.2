<HTML>
<HEAD><TITLE>HotBot results: "Christie Abbott" (19+)</TITLE>
<!-- 'libwww-perl/5.08': ---unknown -->
</HEAD>
<BODY bgcolor="#ffffff" background="/images/bg.base.wide.white.gif" text="#000000" vlink="#ff0000" alink="#0000ff" link="#0000ff"><A HREF="http://help.hotbot.com/?UID=IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A">Help</A> | <A HREF="http://www.wired.com/">WIRED MAGAZINE</A> | <A HREF="http://www.hotwired.com/">HOTWIRED</A> | <A HREF="http://www.wired.com/wired/">WIRED NEWS</A> | <A HREF="http://www.suck.com/">SUCK.COM</A> | <A HREF="http://www.wired.com/home/">W</A><TABLE cellpadding="0" cellspacing="0">
<TR><TD  valign="CENTER" align="LEFT" NOWRAP><A HREF="http://help.hotbot.com/?UID=IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A"><IMG SRC="/images/logo.hb4.white.gif"  BORDER=0 WIDTH=195 HEIGHT=45 ALT="- H o t B o t -"></A></TD>
<TD  valign="CENTER" align="RIGHT" NOWRAP width="380"><A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=6673&_h=I0055gwlJ&_c=6673&_url=http://turbotax.vitabella.com/%3F1305&_slot=mainCobrand"><IMG SRC="http://images.hotbot.com/ads/turbotax/APRIL1251.GIF"  BORDER=0 WIDTH=125 HEIGHT=35 ALT="Click here for TurboTax"></A></TD></TR>
</TABLE>

<P>
<P>
<FORM action="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD></TR>
<TR ><TD  valign="BOTTOM"></TD>
<TD  valign="BOTTOM" align="RIGHT">
look for&nbsp;<SELECT NAME="SM" >
	<OPTION VALUE="MC">all the words
	<OPTION VALUE="SC" selected>any of the words
	<OPTION VALUE="phrase">the exact phrase
	<OPTION VALUE="title">words in the title
	<OPTION VALUE="name">the person
	<OPTION VALUE="url">links to this URL
	<OPTION VALUE="B">the Boolean phrase
</SELECT></TD></TR>
<TR ><TD >
<FONT FACE="geneva, arial" SIZE=2 COLOR=#FFFFFF><B><I>Search: the Web</I></B></FONT><BR>
</TD>
<TD ><INPUT TYPE=text VALUE="&quot;Christie Abbott&quot;" SIZE=39 name="MT" maxlength="300"></TD>
<TD  valign="center"><INPUT TYPE="image" SRC="/images/btn.search-borderless.none.gif" name="act.search" border="0" WIDTH=67 HEIGHT=21 VALUE="Search"></TD></TR>
<TR ><TD ></TD>
<TD  cellspan="2">

</TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<FORM action="http://www.hotbot.com/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/" ><INPUT TYPE="submit" NAME="act.map.newsearch" VALUE="New Search"> | <INPUT TYPE="submit" NAME="act.map.revise" VALUE="Revise Search"><INPUT TYPE="hidden" NAME="MT" VALUE="&quot;Christie Abbott&quot;"><INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=9331&_h=I0055gwlJ&_c=9331&_url=http://security.atbackup.com/%3F1143&_slot=DEF"><IMG SRC="http://images.hotbot.com/ads/atbackup/burdened1CASIEFull.gif"  BORDER=1 WIDTH=468 HEIGHT=60 ALT="@Backup - 30-day FREE trial. Click Here!"></A></TD></TR>
<TR ><TD ></TD>
<TD  align="CENTER"><A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=9331&_h=I0055gwlJ&_c=9331&_url=http://security.atbackup.com/%3F1143&_slot=DEF"><FONT SIZE=2>@Backup - 30-day FREE trial. Click Here!</FONT></A></TD></TR>
</TABLE>


<!-- QUERY:
 (  "Christie Abbott")
 -->
<TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=136 HEIGHT=1 ALT="&nbsp;"></TD>
<TD >
<B>Returned:</B> 21 matches.
</TD></TR>
</TABLE>
<TABLE cellpadding="0" cellspacing="0">
<TR><TD  valign="TOP" align="RIGHT">
<A HREF="http://www.hotbot.com/">The Web</A> <BR> <A HREF="http://www.hotbot.com/usenet.html">Usenet</A> <BR> <A HREF="http://www.hotbot.com/newsbot/index.html">Top News Sites</A> <BR> <A HREF="http://www.hotbot.com/partners/business.html">Businesses</A> <BR> <A HREF="http://www.hotbot.com/partners/people.html">People</A> <BR> <A HREF="http://www.hotbot.com/partners/email.html">Email Addresses</A> <BR> <A HREF="http://www.classifieds2000.com/cgi-cls/display.exe?hotbot+class">Classifieds</A> <BR> <A HREF="http://www.hotbot.com/partners/websitez.html">Domain Names</A> <BR> <A HREF="http://stocks.hotbot.com/">Stocks</A> <BR> <A HREF="http://www.hotbot.com/partners/forumone.html">Discussion Groups</A> <BR> <A HREF="http://www.hotbot.com/partners/filez.html">ShareWare</A>
<BR>
<IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=115 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=15 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><FORM action="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD  width="300" align="LEFT"><B>Breakdown:</B>&nbsp;<FONT size=-1>christie:&nbsp;113118, abbott:&nbsp;139107</font></TD>
<TD ><INPUT TYPE="image" SRC="/images/btn.leftarrow.on.gif" name="act.prev" border="0" WIDTH=16 HEIGHT=19 VALUE="Prev"></TD>
<TD >&nbsp;19&nbsp;-&nbsp;21&nbsp;</TD>
<TD ></TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="MT" VALUE="&quot;Christie Abbott&quot;"><INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<FORM METHOD="get" ACTION="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html" ><INPUT TYPE="hidden" NAME="_id" VALUE="9425"><INPUT TYPE="hidden" NAME="_h" VALUE="I0055gwlJ"><INPUT TYPE="hidden" NAME="_c" VALUE="9425"><INPUT TYPE="hidden" NAME="_url" VALUE="http://www.elibrary.com/s/hotbot"><INPUT TYPE="hidden" NAME="_slot" VALUE="htmlAd"><TABLE cellpadding="0" cellspacing="0" border=0 WIDTH=460>

<TR><TD>

<IMG src="http://static.hotbot.com/images/pix.gif" width=460, height=1>

<UL>

<LI><FONT size="-1" face="verdana, arial, geneva, sans-serif" color="#000000">

<I>Can't find it?  Try

<A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=9425&_h=I0055gwlJ&_c=9425&_slot=htmlAd&_url=http://www.elibrary.com/s/hotbot">HotBot's Research Service</A></a>.</i>

</font>

</ul>

<TR><TD>

<BR>

<IMG src="http://static.hotbot.com/images/pix.gif" width=460, height=1></TD>

</TR>

</TABLE>

</FORM><TABLE cellpadding="0" cellspacing="0">
<TR >
<TD  width="20" align="left" valign="top"><B>19. </B>
</TD>
<TD ><A HREF="http://www.webring.org/index/a.1.html">Webring Index: a</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>88%</FONT></TD>
<TD >Webring Comprehensive Index For a subject-oriented guide to the Webrings, check out RingWorld, our online directory. This index is rebuilt nightly. New additions and changes will not appear here for up to 24 hours. -- a -- 38117 rings total -...<BR>
http://www.webring.org/index/a.1.html, 42595 bytes, 01Mar98</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>20. </B>
</TD>
<TD ><A HREF="http://www.lumal.com/usa/cis/female.html">Agency Summary November 15, 1996 00:56:56</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>74%</FONT></TD>
<TD >BACK TO THE GATE> RANTINGS of a fierce technopath the STANster AGENCY STABLE OF ACTORS Agency unknown or see manager Aida Turturro Alison LaPlaca Allison Folland Amanda Anka Amy Pietz Angela Ortiz Angie Everhart Angie Harmon Anita Barone Ann Anello.<BR>
http://www.lumal.com/usa/cis/female.html, 50758 bytes, 13Oct97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>21. </B>
</TD>
<TD ><A HREF="http://www.xenite.org/kpdoerr/xena_tubewatch90797.html">AL Wheaties Tube Watch</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>61%</FONT></TD>
<TD >UPDATE OF SEPTEMBER 7, 1997 Update of July 18 / TubeWatch Calendar ~Guardian of the Tube~~ Somewhat High Priestess of TV/Movie Trivia A.Y.E.  angela.wheaton@snet.net I wanted to take this opportunity to say how much I am SADDENED by the recent.<BR>
http://www.xenite.org/kpdoerr/xena_tubewatch90797.html, 108169 bytes, 15Oct97</TD></TR>
</TABLE>


<FORM action="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD  width="300" align="LEFT"><B>Breakdown:</B>&nbsp;<FONT size=-1>christie:&nbsp;113118, abbott:&nbsp;139107</font></TD>
<TD ><INPUT TYPE="image" SRC="/images/btn.leftarrow.on.gif" name="act.prev" border="0" WIDTH=16 HEIGHT=19 VALUE="Prev"></TD>
<TD >&nbsp;19&nbsp;-&nbsp;21&nbsp;</TD>
<TD ></TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="MT" VALUE="&quot;Christie Abbott&quot;"><INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
</TD></TR>
</TABLE>
<TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=9331&_h=I0055gwlJ&_c=9331&_url=http://security.atbackup.com/%3F1143&_slot=DEF"><IMG SRC="http://images.hotbot.com/ads/atbackup/burdened1CASIEFull.gif"  BORDER=1 WIDTH=468 HEIGHT=60 ALT="@Backup - 30-day FREE trial. Click Here!"></A></TD></TR>
<TR ><TD ></TD>
<TD  align="CENTER"><A HREF="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/clickthru.html?_id=9331&_h=I0055gwlJ&_c=9331&_url=http://security.atbackup.com/%3F1143&_slot=DEF"><FONT SIZE=2>@Backup - 30-day FREE trial. Click Here!</FONT></A></TD></TR>
</TABLE>

<P>
<FORM action="/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD></TR>
<TR ><TD  valign="BOTTOM"></TD>
<TD  valign="BOTTOM" align="RIGHT">
look for&nbsp;<SELECT NAME="SM" >
	<OPTION VALUE="MC">all the words
	<OPTION VALUE="SC" selected>any of the words
	<OPTION VALUE="phrase">the exact phrase
	<OPTION VALUE="title">words in the title
	<OPTION VALUE="name">the person
	<OPTION VALUE="url">links to this URL
	<OPTION VALUE="B">the Boolean phrase
</SELECT></TD></TR>
<TR ><TD >
<FONT FACE="geneva, arial" SIZE=2 COLOR=#FFFFFF><B><I>Search: the Web</I></B></FONT><BR>
</TD>
<TD ><INPUT TYPE=text VALUE="&quot;Christie Abbott&quot;" SIZE=39 name="MT" maxlength="300"></TD>
<TD  valign="center"><INPUT TYPE="image" SRC="/images/btn.search-borderless.none.gif" name="act.search" border="0" WIDTH=67 HEIGHT=21 VALUE="Search"></TD></TR>
<TR ><TD ></TD>
<TD  cellspan="2">

</TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<FORM action="http://www.hotbot.com/IU1MdRovF9AA574E4967CA78FBDFFA2BF6188B2A/" ><INPUT TYPE="submit" NAME="act.map.newsearch" VALUE="New Search"> | <INPUT TYPE="submit" NAME="act.map.revise" VALUE="Revise Search"><INPUT TYPE="hidden" NAME="MT" VALUE="&quot;Christie Abbott&quot;"><INPUT TYPE="hidden" NAME="base" VALUE="18"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>

<A HREF="http://www.inktomi.com/"><IMG SRC="/images/logo.inktomi.white.gif"  BORDER=0 WIDTH=126 HEIGHT=40 ALT="powered by INKTOMI"></A><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><FONT face="Arial, Helvetica" size=1>
	Copyright &#169; 1996-97 Wired Digital, Inc. All rights reserved.
    </FONT></TD></TR>
</TABLE>
</BODY>
</HTML>
<!-- time = 214738 microseconds per iteration -->


