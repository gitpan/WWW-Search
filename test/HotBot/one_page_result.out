http://www.metroplexweb.com/actingclass/
http://www.palladiumnet.com/WWWishbone/MyShow/TheCast/
http://crisny.org/not-for-profit/cllassie/12tteamb.htm
http://www.sagami.or.jp/~harasawa/seiyuu/List/NishimuraChinami.html
http://pbs.org/wishbone/factsabouttheseries.html
http://www.webmill.com/WebSleuth/celeb/adam_springfield.shtml
http://www3.pbs.org/whatson/1997/10/press/wish__000201.html
http://www.hollywood.com/news/topstories/06-09-98/html/2-5.html
http://149.48.46.5/insidepbs/news/wishbone2.html
http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/WWW/Search/HotBot.html
http://ring.aist.go.jp/archives/lang/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm
http://www.kfki.hu/ftp/packages/perl/CPAN/authors/id/M/MT/MTHURN/HotBot.pm
http://www.gbgm-umc.org/willowhill/news/newslet/newslet.html
http://www.lumal.com/usa/cis/female.html
